app/assets/javascripts/app/discussions/index.html.slim

```diff
@@ -2,10 +2,11 @@ div ui-view="sidebar"
 
 .TopBarLayout
   .TopBarLayout-header
-    md-toolbar.md-whiteframe-z1.Toolbar
```

Comment:
Why isn't the Toolbar class needed anymore? And if so, why did you changed its rules?